src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -466,65 +471,62 @@ function MoneyRequestConfirmationList(props) {
         }
     };
 
-    /**
-     * @param {String} paymentMethod
-     */
     const confirm = useCallback(
-        (paymentMethod) => {
-            if (_.isEmpty(selectedParticipants)) {
+        (paymentMethod: PaymentMethodType | undefined) => {
+            if (!selectedParticipants.length) {
                 return;
             }
-            if (props.iouCategory && props.iouCategory.length > CONST.API_TRANSACTION_CATEGORY_MAX_LENGTH) {
+            if (iouCategory && iouCategory.length > CONST.API_TRANSACTION_CATEGORY_MAX_LENGTH) {
```

Comment:
`iouCategory` has default value
```suggestion
            if (iouCategory.length > CONST.API_TRANSACTION_CATEGORY_MAX_LENGTH) {
```